ehrapy/tools/cohort_tracking/_cohort_tracker.py

```diff
@@ -0,0 +1,332 @@
+        for column in self.columns:
+            if column in self.categorical:
+                # if e.g. a column containing integers is deemed categorical, coerce it to categorical
+                df[column] = df[column].astype("category")
+                column_structure[column] = {category: [] for category in df[column].cat.categories}
+            else:
+                column_structure[column] = []
+
+        return column_structure
+
+    def _get_column_dicts(self, table_one):
+        for col, value in self.track.items():
+            if isinstance(value, dict):
+                self._get_cat_dicts(table_one, col)
+            else:
+                self._get_num_dicts(table_one, col)
+
+    def _get_cat_dicts(self, table_one, col):
+        for cat in self.track[col].keys():
+            # if tableone does not have the category of this column anymore, set the percentage to 0
+            # for categorized columns (e.g. gender 1.0/0.0), str(cat) helps to avoid considering the category as a float
+            if (col, str(cat)) in table_one.cat_table["Overall"].index:
+                pct = float(table_one.cat_table["Overall"].loc[(col, str(cat))].split("(")[1].split(")")[0])
+            else:
+                pct = 0
+            self.track[col][cat].append(pct)
+
+    def _get_num_dicts(self, table_one, col):
+        summary = table_one.cont_table["Overall"].loc[(col, "")]
+        self.track[col].append(summary)
+
+    def reset(self):
+        self.track = self._track_backup
+        self._tracked_steps = 0
+        self._tracked_text = []
+        self._tracked_operations = []
+
+    @property
+    def tracked_steps(self):
+        return self._tracked_steps
+
+    def plot_cohort_change(
+        self,
+        set_axis_labels=True,
+        subfigure_title: bool = False,
+        sns_color_palette: str = "husl",
+        save: str = None,
+        return_plot: bool = False,
+        subplots_kwargs: dict = None,
+        legend_kwargs: dict = None,
+    ):
+        """Plot the cohort change over the tracked steps.
+
+        Create stacked bar plots to monitor cohort changes over the steps tracked with `CohortTracker`.
+
+        Args:
+            set_axis_labels: If `True`, the y-axis labels will be set to the column names.
+            subfigure_title: If `True`, each subplot will have a title with the `label` provided during tracking.
+            sns_color_palette: The color palette to use for the plot. Default is "husl".
+            save: If a string is provided, the plot will be saved to the path specified.
+            return_plot: If `True`, the plot will be returned as a tuple of (fig, ax).
+            subplot_kwargs: Additional keyword arguments for the subplots.
+            legend_kwargs: Additional keyword arguments for the legend.
+
+        Returns:
+            If `return_plot` a :class:`~matplotlib.figure.Figure` and a :class:`~matplotlib.axes.Axes` or a list of it.
+
+        Example:
```

Comment:
```suggestion
        Examples:
```

Just for consistency